backend/test/authorizer/test_authorizer.py

```diff
@@ -134,6 +134,66 @@ def mock_event(
     }
 
 
+def generate_test_config(config_scope: str = "global", is_project: bool = False, admin_only: bool = False) -> dict:
```

Comment:
Again, this feels like something that should be put in a test util as I would expect the config to be used across multiple tests in the near future. I would default it to what we expect to be the most common settings and then have a way to input overrides for the default values. This feels like something that shouldn't be localized to the authorizer.

Reply:
Yeah there's actually already one or two similar ones in other tests, although from looking at other tests I haven't ever seen us use a common utils test-data class, I wonder if there's a reason for that... will look into it.

Reply:
It doesn't seem like we can import things from other test classes since they're not in scope, so I'm going to leave this as-is (unless you're aware of a work around, I wasn't able to find us doing anything like this elsewhere)